{"uuid": "85477d00-5d93-40a2-9525-6767cf35f38f", "children": ["2335dd09-1e56-4a9d-a0e7-0d77c401abd2"], "befores": [{"name": "init_driver", "status": "broken", "statusDetails": {"message": "UnboundLocalError: local variable 'web_driver' referenced before assignment\n", "trace": "  File \"C:\\Users\\SauravSharma\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\SauravSharma\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\_pytest\\fixtures.py\", line 1111, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\SauravSharma\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\_pytest\\fixtures.py\", line 883, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Users\\SauravSharma\\Pytest\\Pom-Pytest\\Tests\\conftest.py\", line 12, in init_driver\n    request.cls.driver = web_driver\n"}, "start": 1648475214511, "stop": 1648475214511}], "start": 1648475214511, "stop": 1648475214694}